src/Conversation/messageBlockHelpers.ts

```diff
@@ -0,0 +1,13 @@
+import * as vscode from "vscode";
+import { MessageBlock } from "../ChatModel/ChatModel";
+
+export function stringToMessageBlocks(text: string): MessageBlock[] {
+  vscode.window
+    .createOutputChannel("vscode-byolad: Response Content")
+    .appendLine(text); // TODO: DELETE ME
+  return JSON.parse(text) as MessageBlock[]; // TODO: Change to parsing the markdown response as discussed in class
```

Comment:
TODOs here

Reply:
Deleted lines 5-7 but will leave the others here to be done in another PR.